diff -Npur steamos-manager-v26.0.1.orig/data/system/reset-oneshot-boot.conf steamos-manager-v26.0.1/data/system/reset-oneshot-boot.conf
--- steamos-manager-v26.0.1.orig/data/system/reset-oneshot-boot.conf	2026-03-05 05:25:34.000000000 +0100
+++ steamos-manager-v26.0.1/data/system/reset-oneshot-boot.conf	2026-03-19 07:56:55.375903418 +0100
@@ -2,5 +2,4 @@
 # This is a failsafe to reset the login mode back to default after a reboot
 # It shouldn't be strictly needed
 # It's a fallback in case a login fails between boot and starting
-ExecStartPre=-rm /etc/sddm.conf.d/zzt-steamos-temp-login.conf
-ExecStartPre=-rm /etc/sddm.conf.d/zzt-holo-temp-login.conf
+ExecStartPre=-rm /etc/sddm.conf.d/zzt-aerynos-temp-login.conf
diff -Npur steamos-manager-v26.0.1.orig/steamos-manager/src/session.rs steamos-manager-v26.0.1/steamos-manager/src/session.rs
--- steamos-manager-v26.0.1.orig/steamos-manager/src/session.rs	2026-03-05 05:25:34.000000000 +0100
+++ steamos-manager-v26.0.1/steamos-manager/src/session.rs	2026-03-19 08:15:45.160356577 +0100
@@ -34,12 +34,12 @@ use crate::{Service, path};
 
 const CONFIG_PREFIX_USR: &str = "/usr/lib/sddm/sddm.conf.d";
 const CONFIG_PREFIX: &str = "/etc/sddm.conf.d";
-const SESSION_CHECK_PATH_LEGACY: &str = "steamos.conf";
-const CONFIG_PATH_LEGACY: &str = "zz-steamos-autologin.conf";
-const TEMPORARY_CONFIG_PATH_LEGACY: &str = "zzt-steamos-temp-login.conf";
-const SESSION_CHECK_PATH: &str = "holo.conf";
-const CONFIG_PATH: &str = "zz-holo-autologin.conf";
-const TEMPORARY_CONFIG_PATH: &str = "zzt-holo-temp-login.conf";
+const SESSION_CHECK_PATH_LEGACY: &str = "aerynos-handheld.conf";
+const CONFIG_PATH_LEGACY: &str = "zz-aerynos-autologin.conf";
+const TEMPORARY_CONFIG_PATH_LEGACY: &str = "zzt-aerynos-temp-login.conf";
+const SESSION_CHECK_PATH: &str = "aerynos-handheld.conf";
+const CONFIG_PATH: &str = "zz-aerynos-autologin.conf";
+const TEMPORARY_CONFIG_PATH: &str = "zzt-aerynos-temp-login.conf";
 
 #[cfg(not(test))]
 static CONFIG_PATHS: OnceCell<ConfigPaths> = OnceCell::const_new();
