diff --git a/etc/meson.build b/etc/meson.build
index 4dc644e..5c9823a 100644
--- a/etc/meson.build
+++ b/etc/meson.build
@@ -1,2 +1 @@
-subdir('UPower.conf.d')
-install_data('UPower.conf', install_dir: get_option('sysconfdir') / 'UPower')
+install_data('UPower.conf', install_dir: get_option('datadir') / 'defaults/UPower')
diff --git a/meson.build b/meson.build
index 2c7018f..8c547da 100644
--- a/meson.build
+++ b/meson.build
@@ -29,6 +29,7 @@ cdata.set_quoted('GETTEXT_PACKAGE', meson.project_name())
 cdata.set_quoted('PACKAGE_VERSION', meson.project_version())
 cdata.set_quoted('VERSION', meson.project_version())
 cdata.set_quoted('PACKAGE_SYSCONF_DIR', get_option('sysconfdir'))
+cdata.set_quoted('PACKAGE_DATA_DIR', get_option('datadir'))
 
 glib_min_version = '2.76'
 
diff --git a/src/up-config.c b/src/up-config.c
index fa4c717..a7482f9 100644
--- a/src/up-config.c
+++ b/src/up-config.c
@@ -261,7 +261,7 @@ up_config_init (UpConfig *config)
 
 	filename = g_strdup (g_getenv ("UPOWER_CONF_FILE_NAME"));
 	if (filename == NULL) {
-		filename = g_build_filename (PACKAGE_SYSCONF_DIR,"UPower", "UPower.conf", NULL);
+		filename = g_build_filename ("usr", PACKAGE_DATA_DIR, "defaults", "UPower", "UPower.conf", NULL);
 		conf_d_path = g_build_filename (PACKAGE_SYSCONF_DIR, "UPower", "UPower.conf.d", NULL);
 	} else {
 		conf_dir = g_path_get_dirname (filename);
