diff --git a/src/main.cpp b/src/main.cpp
index 8c21e68..f2222b2 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -20,6 +20,7 @@
 #include <cerrno>
 #include <csignal>
 
+#include <filesystem>
 #include <iostream>
 #include <string_view>
 
@@ -130,10 +131,19 @@ int main(const int argc, const char **argv) {
     return EXIT_SUCCESS;
   }
 
+  std::error_code   ec;
   const std::string config_dir_path =
       get_env("ANANICY_CPP_CONFDIR").value_or("/etc/ananicy.d");
+      get_env("ANANICY_CPP_CONFDIR")
+            .value_or(std::filesystem::exists("/etc/ananicy.d", ec)
+                         ? "/etc/ananicy.d"
+                         : "/usr/share/defaults/ananicy.d");
   const std::string config_path =
       get_env("ANANICY_CPP_CONF").value_or("/etc/ananicy.d/ananicy.conf");
+      get_env("ANANICY_CPP_CONF")
+            .value_or(std::filesystem::exists("/etc/ananicy.d/ananicy.conf", ec)
+                         ? "/etc/ananicy.d/ananicy.conf"
+                         : "/usr/share/defaults/ananicy.d/ananicy.conf");
 
   Config conf(config_path);
   conf.show();
