diff -Npur OpenGamepadUI-main.orig/rootfs/usr/bin/opengamepadui OpenGamepadUI-main/rootfs/usr/bin/opengamepadui
--- OpenGamepadUI-main.orig/rootfs/usr/bin/opengamepadui	2024-12-30 14:35:31.000000000 +0700
+++ OpenGamepadUI-main/rootfs/usr/bin/opengamepadui	2025-01-02 11:42:48.393449376 +0700
@@ -1,6 +1,5 @@
 #!/bin/bash
-SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
-PREFIX=$(dirname -- "${SCRIPT_DIR}")
+PREFIX=/usr
 ARCH=$(uname -m)
 
 OGUI_BIN=${OGUI_BIN:-"${PREFIX}/share/opengamepadui/opengamepad-ui.${ARCH}"}
