diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -141,5 +141,5 @@ desktop_install_dir = join_paths(prefix, 'share/wayland-sessions')
 portal_install_dir = join_paths(prefix, 'share/xdg-desktop-portal')
 install_data('assets/mango.desktop', install_dir : desktop_install_dir)
 install_data('assets/mango-portals.conf', install_dir : portal_install_dir)
-install_data('assets/config.conf', install_dir : join_paths(sysconfdir, 'mango'))
+install_data('assets/config.conf', install_dir : join_paths('/usr/share/defaults', 'mango'))
 install_data('mmsg/mmsg.1', install_dir : join_paths(mandir, 'man1'))
diff --git a/src/config/parse_config.h b/src/config/parse_config.h
--- a/src/config/parse_config.h
+++ b/src/config/parse_config.h
@@ -7,6 +7,10 @@
 #define SYSCONFDIR "/etc"
 #endif
 
+#ifndef VENDORDIR
+#define VENDORDIR "/usr/share/defaults"
+#endif
+
 // 整数版本 - 截断小数部分
 #define CLAMP_INT(x, min, max)                                                 \
 	((int32_t)(x) < (int32_t)(min)                                             \
@@ -3880,6 +3884,10 @@ bool parse_config(void) {
 			// 如果文件不存在，则使用 /etc/mango/config.conf
 			snprintf(filename, sizeof(filename), "%s/mango/config.conf",
 					 SYSCONFDIR);
+			if (access(filename, F_OK) != 0) {
+				snprintf(filename, sizeof(filename), "%s/mango/config.conf",
+						 VENDORDIR);
+			}
 		}
 	}
 
